webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -789,6 +790,7 @@ export const GlideTable: React.FC<GlideTableProps> = ({
           onColumnMoved={onColumnMoved}
           onColumnResize={onColumnResize}
           onHeaderClicked={onHeaderClicked}
```

Comment:
Do you know the difference between `onHeaderClicked` and `onHeaderContextMenu`?
[onHeaderClicked doc](https://grid.glideapps.com/docs/interfaces/DataEditorProps.html#onHeaderClicked) and [onHeaderContextMenu doc](https://grid.glideapps.com/docs/interfaces/DataEditorProps.html#onHeaderContextMenu) say the same things, and `onHeaderClicked` is supposed to support `right click`.

Reply:
I think that the docs are wrong. In https://github.com/glideapps/glide-data-grid/blob/main/packages/core/src/data-editor/data-editor.tsx , onHeaderClicked is called in two locations times; one for touch events, one `if (args.button === 0 && col === lastMouseDownCol && row === lastMouseDownRow) {`

Reply:
Hmm, prob adding comment would be nice to let the future coder know `onHeaderContextMenu` func is for `right click`